quickstarts/Models.ipynb

```diff
@@ -11,7 +11,7 @@
     },
```

Comment:
<p>Heading is good - the paragraph text is a little unclear, how about this:</p><p> </p><blockquote>Get model</blockquote><blockquote>Use <code>get_model()</code> to retrieve the specific details of a model. You can iterate over all available models using <code>list_models()</code>, but if you already know the model name you can retrieve it directly with <code>get_model()</code> .</blockquote><p> </p><p>I think your intent was to describe the function, then contrast it to <code>list_models</code>, so that's what I wrote. Let me know if I misunderstood your intent. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Thanks Mark, updated the paragraph as per your suggestion.